operator/pkg/apis/core/v1beta1/configconnector_types.go

```diff
@@ -66,6 +66,12 @@ type ConfigConnectorSpec struct {
 	//+kubebuilder:validation:Enum=Absent;Merge
 	//+kubebuilder:validation:Optional
 	StateIntoSpec *StateIntoSpecValue `json:"stateIntoSpec,omitempty"`
+
+	// Experiments are a list of strings that allow opting-in to previews/experiments.
+	// This functionality is generally not supported and not expected to work going forwards,
+	// the intention is to allow new functionality to be delivered to non-production environments quickly,
+	// proving the functionality before it is baked into the supported API.
+	Experiments []string `json:"experiments,omitempty"`
```

Comment:
Is it possible to validate the experiments using Kubebuilder validation (`+kubebuilder:validation`) or CEL (`+kubebuilder:validation:XValidation`)?

Reply:
sure what kind of validation would you like to see? we can also add it on as a follow up 😛 

I think as we are prototypying this we can keep the validations a bit loose at first

Reply:
I was thinking we could validate the experiment names early in CRD (as opposed to in controller). But agreed we could do a follow up.